src/classes/UTIL_DMLService_TEST.cls

```diff
@@ -0,0 +1,699 @@
+    
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS 
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE 
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, 
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, 
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; 
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER 
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT 
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN 
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+
+/**
+* @author Salesforce.org
+* @date 2017
+* @group Utilities
+* @description Service Class to handle DML actions.
+*/
+
+@isTest
+public with sharing class UTIL_DMLService_TEST {
+    
+    /*********************************************************************************************************
+    * @description If you only want to run a single test in this class, fill in its name here.
+    * If you want to run all tests, then use '*'
+    */
+    private static string strTestOnly = '*';
+    
+
+    /*********************************************************************************************************
+    * @description Tests insert of a record with default DMLOption settings.
+    * verify:
+    *   Record inserted
+    * @return void
+    **********************************************************************************************************/
+    private static testMethod void testRecordInsertWithDefaults() {
+        if (strTestOnly != '*' && strTestOnly != 'testRecordInsertWithDefaults') return;
+        
+        Account acct = new Account(Name = 'DMLOptionsAccount');
+        insert acct;
+
+        Contact con = new Contact(FirstName = 'Blue', LastName = 'Cloud', AccountId = acct.Id); 
+                 
+        Test.startTest();
+        UTIL_DMLService.insertRecord(con);
+        Test.stopTest();
+
+        List<Contact> retrievedContacts = [SELECT Id, FirstName, LastName, AccountId 
+                                           FROM Contact];
+
+        if (!retrievedContacts.isEmpty()) {
```

Comment:
shouldn't this test assert that there was a retrieved contact, rather than gracefully handling it if it failed?  that's the whole point of testing your new insertRecord() method.